packages/eslint-plugin-ecmascript-compat/lib/rule.js

```diff
@@ -1,21 +1,36 @@
 const compatibility = require('./compatibility');
 const { createDelegatee, delegatingVisitor } = require('./delegation');
 const features = require('./features');
+const polyfilling = require('./polyfilling');
 const targetRuntimes = require('./targetRuntimes');
 
 const targets = targetRuntimes();
-const delegateeConfigs = compatibility
-  .forbiddenFeatures(features, targets)
-  .map((feature) => feature.ruleConfig);
+const forbiddenFeatures = compatibility.forbiddenFeatures(features, targets);
```

Comment:
I think we should rename this function and the var, because it's not necessarily forbidden now.

I suggest `unsupportedFeatures`